### PR TITLE
Create Zoom Data Models and add to Standard Detections

### PR DESCRIPTION
**Pull request by bseb**
### Background
Add Content for Zoom log puller

### Changes

Create Zoom Data Models
Add Zoom log types to standard detections

Apologies that this PR is kinda huge
### Testing

* Unit and CI/CD

### CODE REVIEW

File: zoom_operation_rules/zoom_operation_user_granted_admin.py
Comment (bseb) [inline zoom_operation_rules/zoom_operation_user_granted_admin.py:1]: I did not use the Standard admin access rule here because the Zoom data model does not contain all the fields that rule requires

**Comment by bseb** [comment]
I'll have a few more detections coming, but I figure this PR is pretty big already so I'll push the other changes in a separate PR

**Review by darwayne — APPROVED** [review]

File: global_helpers/panther_event_type_helpers.py
Comment (lindsey-w) [inline global_helpers/panther_event_type_helpers.py:4]: for naming consistency with the `*_LOGIN` would it be worth it to make this `SUCCESSFUL_LOGOUT`? 🤷🏻‍♀️

File: global_helpers/panther_event_type_helpers.py
Comment (lindsey-w) [inline global_helpers/panther_event_type_helpers.py:8]: This has nothing to do with this PR, but could we add a comment to distinguish what the difference between these are?

File: global_helpers/panther_zoom_helpers.py
Comment (lindsey-w) [inline global_helpers/panther_zoom_helpers.py:15]: are there any cases where the sizes of the resulting array would cause these to raise an exception? Another option is `re.compile()`, but maybe that isn't necessary

File: zoom_operation_rules/zoom_operation_passcode_disabled.py
Comment (lindsey-w) [inline zoom_operation_rules/zoom_operation_passcode_disabled.py:11]:
```suggestion
    return "Passcode" in context["Change"] and context["DisabledSetting"]:
```

File: zoom_operation_rules/zoom_operation_user_granted_admin.py
Comment (lindsey-w) [inline zoom_operation_rules/zoom_operation_user_granted_admin.py:12]:
```suggestion
    return "Member to Admin" in context["Change"]:
```

File: zoom_operation_rules/zoom_operation_user_granted_admin.py
Comment (lindsey-w) [inline zoom_operation_rules/zoom_operation_user_granted_admin.py:5]: should this be... 
```suggestion
    if event.get("Action") != "Update" or event.get("category_type") != "User":
```

**Review by lindsey-w — COMMENTED** [review]
Nice work! Some interesting extractions we have to do here.  

I had just a few questions:
- will a follow on PR add a zoom pack?
- should we add zoom to the standard pack (data model + new global + brute force login)

File: global_helpers/panther_event_type_helpers.py
Comment (bseb) [reply global_helpers/panther_event_type_helpers.py:8]: Yup, good call.

File: global_helpers/panther_event_type_helpers.py
Comment (bseb) [reply global_helpers/panther_event_type_helpers.py:4]: I'm torn here.  I like the consistency but it makes me wonder what an Unsuccessful logout would be

File: zoom_operation_rules/zoom_operation_user_granted_admin.py
Comment (bseb) [reply zoom_operation_rules/zoom_operation_user_granted_admin.py:12]: Much cleaner, thank you. I always forget to go back and clean up stuff like this

File: global_helpers/panther_event_type_helpers.py
Comment (lindsey-w) [reply global_helpers/panther_event_type_helpers.py:4]: I've been in a state where I can't logout from an application before, but it is rare (and the application probably didn't log the fact that is was unsuccessful).  Your call

**Comment by bseb** [comment]
> will a follow on PR add a zoom pack?
> should we add zoom to the standard pack (data model + new global + brute force login) 

That is my plan, I was going to bounce the same questions off of you. That PR will be forthcoming after this one merges

File: global_helpers/panther_event_type_helpers.py
Comment (bseb) [reply global_helpers/panther_event_type_helpers.py:4]: Yeah I think we should do it after pondering it for a few more minutes with a cup of coffee. There could be a situation down the road where we find that log event and better to not have to change every instance of this field

File: global_helpers/panther_zoom_helpers.py
Comment (bseb) [reply global_helpers/panther_zoom_helpers.py:15]: From what I have seen of the logs the sentence structure of this field is consistent among events that share a category and type so I think we should be OK.

That being said, this particular field in the Zoom logs feels very brittle to me. I had some conversation with Hakmiller about seeing if team ingestion can perhaps do some more extraction from this field upon ingestion. I am going to have that conversation and hopefully update / retire these helpers in the semi-near future.

**Review by darwayne — APPROVED** [review]

File: indicator_creation_rules/new_user_account_logging.py
Comment (wey-chiang) [inline indicator_creation_rules/new_user_account_logging.py:16]: nit: standardize missing fields

```suggestion
    new_account = event.udm("user_account_id") or "<UNKNOWN_ACCOUNT>"
```

File: indicator_creation_rules/new_user_account_logging.py
Comment (wey-chiang) [inline indicator_creation_rules/new_user_account_logging.py:28]: nit: standardize missing fields

```suggestion
    return f"A new user account was created - [{event.udm('user') or '<UNKNOWN_USER>'}]"
```

**Review by wey-chiang — APPROVED** [review]
LGTM, nice work!

**Review by darwayne — APPROVED** [review]

File: global_helpers/panther_event_type_helpers.py
Comment (lindsey-w) [inline global_helpers/panther_event_type_helpers.py:9]: 💯

**Review by lindsey-w — APPROVED** [review]
Nice work!

**Pull request merged**
